solutions/swb-reference/src/HPC/README.md

```diff
@@ -30,20 +30,19 @@ Scheduling:
       Networking:
         SubnetIds:
           - subnet-abc123
-Region: us-west-2
+Region: us-east-1
 Image:
   Os: alinux2
 ```
-4. Create atleast one S3 bucket on your AWS Main Account containing files necessary to complete a job such as scripts and data. Add the `S3 Full Access` policy to the IAM role assumed by the head node of your cluster to have permission to execute S3 actions. Lastly, similar to the policy below, attach the appropriate bucket policy to your S3 bucket to allow your cluster head node to access the bucket.
-
-```
+5. Create atleast one S3 bucket on your AWS Main Account containing files necessary to complete a job such as scripts and data. Attach the `S3 Full Access` policy to both the IAM role of the head node (`clusterName-RoleHeadNode`) and the IAM role of the compute nodes (`clusterName-Role`) of your ParallelCluster for the nodes to execute S3 operations. Lastly, similar to the policy below, attach the appropriate bucket policy to your S3 bucket to allow your ParallelCluster to access the bucket.
```

Comment:
Let's format it like this `<clusterName>-RoleHeadNode` and mention it's a prefix. Also mention that these roles are in the hosting account